src/gui/reports/ReportsWidgetHealthcheck.ui

```diff
@@ -54,6 +54,16 @@
      </attribute>
     </widget>
    </item>
+   <item>
+    <widget class="QCheckBox" name="showWeakOnly">
+     <property name="text">
+      <string>Show weak passwords only</string>
+     </property>
+     <property name="checked">
+       <bool>true</bool>
```

Comment:
Maybe this shouldn't be checked by default?

Reply:
I'm fine with unchecking it. I only set it to checked to maintain the current behavior.

If you're doing things right, then most of your passwords will be good. Having this default to unchecked you would see a lot more passwords and possibly more information than you need by default.